src/pages/register.tsx

```diff
@@ -0,0 +1,152 @@
+import React from "react";
+import {
+  Box,
+  Typography,
+  TextField,
+  Stack,
+  Button,
+  Select,
+  MenuItem,
+  SelectChangeEvent,
+  Grid,
+  InputAdornment,
+} from "@mui/material";
+import PasswordTextFeild from "@/components/public/PasswordTextField";
+import Logo from "@/components/public/Logo";
+import { Gender } from "enum/gender";
+import { Dayjs } from "dayjs";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import CommonTextField from "@/components/public/CommonTextField";
+import CommonDropdown from "@/components/public/CommonDropdown";
+import { MobileDatePicker } from "@mui/x-date-pickers";
+import CalendarTodayIcon from "@mui/icons-material/CalendarToday";
+
+export default function Home() {
+  const [email, setEmail] = React.useState("");
+  const [password, setPassword] = React.useState("");
+  const [confirmPassword, setConfirmPassword] = React.useState("");
+  // const [isSubmit, setIsSubmit] = React.useState(false);
+  const [value, setValue] = React.useState<Dayjs | null>(null);
+  const [displayName, setDisplayName] = React.useState<string>("");
+  const [gender, setGender] = React.useState<string>("");
+
+  const handleDisplayNameChange = (
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void => {
+    setDisplayName(event.target.value);
+  };
+
+  function handleEmailChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setEmail(event.target.value);
+  }
+
+  function handlePasswordChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setPassword(event.target.value);
+  }
+
+  function handleConfirmPasswordChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setConfirmPassword(event.target.value);
+  }
+
+  function handleGenderChange(event: SelectChangeEvent<string>): void {
+    setGender(event.target.value);
+  }
+
+  const handleSelectChange = (event: SelectChangeEvent): void => {
+    setGender(event.target.value as string);
+  };
```

Comment:
Are the purposes of these functions the same thing?
If they are duplicated, please delete the unused one.

Reply:
Already delete the unused one.

Reply:
👍🏻